src/main/java/net/minecraftforge/client/GuiIngameForge.java

```diff
@@ -83,6 +87,9 @@ public class GuiIngameForge extends GuiIngame
     public static boolean renderFood = true;
     public static boolean renderHealthMount = true;
     public static boolean renderAir = true;
+    /**
+     * experiAnce?
```

Comment:
Unneeded javadoc

Reply:
It's intended for making devs aware that this field actually refers to `experience`. I've added some more doc to it, should be better now.